services/federator/test/unit/Test/Federator/ExternalServer.hs

```diff
@@ -48,27 +48,27 @@ tests =
 requestBrigSuccess :: TestTree
 requestBrigSuccess =
   testCase "should translate response from brig to 'InwardResponseBody' when response has status 200" $
-    runM . evalMock @Brig @IO $ do
-      mockBrigCallReturns @IO (\_ _ _ -> pure (HTTP.ok200, Just "response body"))
+    runM . evalMock @Service @IO $ do
+      mockServiceCallReturns @IO (\_ _ _ _ -> pure (HTTP.ok200, Just "response body"))
       let request = Request Brig "/users" "\"foo\"" exampleDomain
 
-      res :: InwardResponse <- mock @Brig @IO . noLogs . Polysemy.runReader allowAllSettings $ callLocal request
-      actualCalls <- mockBrigCallCalls @IO
-      let expectedCall = ("/users", "\"foo\"", aValidDomain)
+      res :: InwardResponse <- mock @Service @IO . noLogs . Polysemy.runReader allowAllSettings $ callLocal request
+      actualCalls <- mockServiceCallCalls @IO
+      let expectedCall = (Brig, "/users", "\"foo\"", aValidDomain)
       embed $ assertEqual "one call to brig should be made" [expectedCall] actualCalls
       embed $ assertEqual "response should be success with correct body" (InwardResponseBody "response body") res
 
 requestBrigFailure :: TestTree
 requestBrigFailure =
   testCase "should translate response from brig to 'InwardResponseBody' when response has status 404" $
-    runM . evalMock @Brig @IO $ do
-      mockBrigCallReturns @IO (\_ _ _ -> pure (HTTP.notFound404, Just "response body"))
+    runM . evalMock @Service @IO $ do
+      mockServiceCallReturns @IO (\_ _ _ _ -> pure (HTTP.notFound404, Just "response body"))
```

Comment:
Where does this `mockServiceCallReturns` thing come from? I can't find it anywhere in the codebase.

Reply:
It's automatically generated by `genMock`.